odl/util/testutils.py

```diff
@@ -648,6 +648,11 @@ def test(arguments=None):
         raise ImportError('ODL tests cannot be run without `pytest` installed.'
                           '\nRun `$ pip install [--user] odl[testing]` in '
                           'order to install `pytest`.')
+    if pytest.__version__ in ['3.7.0', '3.7.1']:
```

Comment:
This doesn't cover the same range as the pip requirements (not 3.7.2 and not the possibly broken subsequent versions).
Suggestion:
```python
from packaging import version
...
# TODO(sbanert): adapt constraint when upstream issue is fixed
# See <link to pytest issue>
if not version.parse(pytest.__version__) < version.parse("3.7*"):
    raise RuntimeError(...)
```
This uses the faster-to-import `packaging.version`, and it includes dev versions like 3.7.0dev0.

Reply:
I mean faster than `pkg_resources.parse_version`. The dependency on `packaging` has been added recently.